packages/fxa-auth-server/test/local/google-maps-services.js

```diff
@@ -132,12 +136,34 @@ describe('GoogleMapsServices', () => {
       );
     });
 
-    it('Throws Error for invalid country code', async () => {
-      const expectedMessage =
-        'Invalid country (Germany). Only ISO 3166-1 alpha-2 country codes are supported.';
+    it('returns location for zip code and country if more than 1 result is returned with matching states', async () => {
+      const geocodeResultManyMatchingStates = deepCopy(geocodeResultMany);
+      geocodeResultManyMatchingStates.data.results[1].address_components[0].short_name =
+        'MD';
+      const expectedResult = 'MD';
+      const expectedAddress = '11111, United States of America';
+      googleClient.geocode = sinon
+        .stub()
+        .resolves(geocodeResultManyMatchingStates);
+
+      const actualResult = await googleMapsServices.getStateFromZip(
+        '11111',
+        'US'
+      );
+      assert.equal(actualResult, expectedResult);
+      assert.isTrue(googleClient.geocode.calledOnce);
+      assert.equal(
+        googleClient.geocode.getCall(0).args[0].params.address,
+        expectedAddress
+      );
+    });
+
+    it('Throws error if more than 1 result is returned with mismatching states', async () => {
+      const expectedMessage = 'Could not find unique results. (11111, Germany)';
+      googleClient.geocode = sinon.stub().resolves(geocodeResultMany);
 
       try {
-        await googleMapsServices.getStateFromZip('11111', 'Germany');
+        await googleMapsServices.getStateFromZip('11111', 'DE');
```

Comment:
Some of the edits look weird here, because I moved the previous "multiple `results`" test case up to be adjacent to the new test case.

While I'm here, I can update the arguments to be consistent if desired, but I don't think it matters for the error test cases in question.